llvm/include/llvm/Target/TargetSelectionDAG.td

```diff
@@ -353,6 +353,8 @@ class SDNode<string opcode, SDTypeProfile typeprof,
   string SDClass = sdclass;
   let Properties = props;
   SDTypeProfile TypeProfile = typeprof;
+  bit IsStrictFP = false;
+  bits<64> TSFlags = 0;
```

Comment:
TSFlags in SDNodeDesc in #119709 is declared as uint32_t. Are you going to enlarge that or should this be 32?

Reply:
Thanks for spotting this. I think 32 bits should be more than enough, at least for now.